CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# v0.4.1 (06.02.2022)
```

Comment:
We should probably create an UNRELEASED section, and update this after the official release has been cut.